Airports/EGCC/SMR/Regions.txt

```diff
@@ -1,3 +1,10 @@
+;Outer Border
+REGIONNAME Manchester
+7031632 N053.20.48.752 W002.27.21.159
```

Comment:
For now, change the colour to whatever the inactive sector background is set to for the Manchester Combined profile - you can get this from the settings file for symbology that this profile points to. But @tasosb @cpawley leave #1297 open and I’ll double check where this went wron (i.e. the background was removed and what colour it used to be / should be) when I have access to EuroScope